dnf.spec

```diff
@@ -147,13 +147,11 @@ Common data and configuration files for DNF
 %package -n %{yum_subpackage_name}
 Requires:       %{name} = %{version}-%{release}
 Summary:        %{pkg_summary}
-%if 0%{?fedora}
-%if 0%{?fedora} >= 31
-Conflicts:      yum
-%else
+%if 0%{?fedora} && 0%{?fedora} < 31
```

Comment:
This is wrong. You want `0%{?fedora} >= 31 || 0%{?rhel} >= 8` since it's supposed to kick in for both cases.

Reply:
I don't think we need this to kick in for rhel-8. After all, it wouldn't before the patch, either.

Reply:
Hmm, true. The subpackage was called `yum4` and `nextgen-yum4` in RHEL 7. :/

Reply:
Yup :) But thanks for pointing out this particular condition; I just realized it should actually be the reverse! What it aims to achieve is that we continue providing the `dnf-yum` name even in Fedora 31 and later. So it should rather say: `%if 0%{?fedora} && 0%{?fedora} >= 31`

Reply:
You only need `%if 0%{?fedora} >= 31`, as that implies that `%fedora` is defined and nonzero.

Reply:
True!